test-case/verify-sof-firmware-load.sh

```diff
@@ -35,7 +35,8 @@ if $cmd | grep -q " sof-audio.*Firmware.*version"; then
     exit 0
 else
     journalctl --dmesg --lines 50
-    journalctl --dmesg | grep -C 1 -i version
-    printf '$cmd was %s\n' "$cmd"
+    journalctl --dmesg | grep -q " sof-audio.*Firmware.*version";
```

Comment:
I don't understand the `-q` and the lack of `|| true`, did you test this locally? It looks this was just copied from above.